src/components/HomePage/HomePage.tsx

```diff
@@ -1,17 +1,25 @@
 import React from "react"
+import { useState } from "react"
 import * as S from "./HomePage.styles"
+import { Search } from "../Search"
 import { RatingHistoryChart } from "components/RatingHistoryChart"
 import { fakeData } from "./"
 import { parsePoints } from "common/utils"
+import { Player } from "components/Search/types"
 
 export const HomePage: React.FC = () => {
+  const [players, setPlayers] = useState<Player[] | unknown>([])
```

Comment:
can stay for now, but we will have to think about storing errors in a different state than player stats